source/NVDAObjects/IAccessible/chromium.py

```diff
@@ -185,11 +187,35 @@ class Editor(ia2Web.Editor):
 	TextInfo = EditorTextInfo
 
 
+class RedirectDocument(ia2Web.Ia2Web):
+	"""The NVDAObject for the redirect document object in Electron."""
+
+	def objectFromPointRedirect(self, x: int, y: int):
+		docObj: Document = self.previous.lastChild
+		redirect = docObj.IAccessibleObject.accHitTest(x, y)
+
+		if not redirect:
+			return None
+
+		redirect = IAccessibleHandler.normalizeIAccessible(redirect)
+		obj: IAccessible = IAccessible(IAccessibleObject=redirect, IAccessibleChildID=winUser.CHILDID_SELF)
+
+		return obj
+
+
 def findExtraOverlayClasses(obj, clsList):
 	"""Determine the most appropriate class(es) for Chromium objects.
 	This works similarly to L{NVDAObjects.NVDAObject.findOverlayClasses} except that it never calls any other findOverlayClasses method.
 	"""
 	if (
+		obj.role == controlTypes.Role.PANE
+		and obj.childCount == 0
+		and obj.previous
+		and obj.previous.childCount != 0
+		and obj.previous.lastChild.windowClassName == "Chrome_RenderWidgetHostHWND"
```

Comment:
I'm honestly afraid that this will be a bit expensive here and there. It should at least be changed to:

```suggestion
		and obj.previous.lastChild
		and obj.previous.lastChild.windowClassName == "Chrome_RenderWidgetHostHWND"
```

Reply:
```
		and obj.previous.childCount != 0
```

So can this be removed